cpp/marc_tools/marc_field_copy_for_remit_purposes.cc

```diff
@@ -0,0 +1,88 @@
+/** \brief A tool for adding keywords extracted from titles to MARC records.
```

Comment:
This program is specific to remid, and if we have to do some additional changes in the data, we will also put it in there. This means:
1) The file should be moved from cpp/marc_tools tp cpp/harvest
2) It doesn't make much sense to call it something like "copy" in the name, please keep it more generic, e.g. `marc_field_copy_for_remit_purposes.cc` => `remid.cc` or `remid_augment.cc`
3) the \brief section seems to be a copy from another program, please change it to something more appropriate